open-machine-learning-jupyter-book/assignments/data-science/estimation-of-COVID-19-pandemic.ipynb

```diff
@@ -13,10 +13,25 @@
   },
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #11.    class Testget_Onecolumn_groupby_sum(unittest.TestCase):
</pre><p>The class name must be in camel style. Please follow the example I shared to format your code.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/open-academy/machine-learning/pull/156/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='15'/>

Reply:
<p>Modified</p>